packages/smarthr-ui/src/components/Header/AppLauncher/AppLauncher.tsx

```diff
@@ -69,75 +68,134 @@ export const AppLauncher: React.FC<Props & ElementProps> = ({
   enableNew,
   ...props
 }) => {
-  const triggerLabel = useMemo(
-    () => decorators?.triggerLabel?.(TRIGGER_LABEL) || TRIGGER_LABEL,
-    [decorators],
-  )
+  const calculatedApps = useMemo(() => {
+    const result: {
+      base: Props['apps'][number] | undefined
+      others: Props['apps']
+    } = { base: undefined, others: [] }
+
+    apps.forEach((app) => {
+      if (app.type === 'base') {
+        result.base = app
+      } else {
+        result.others.push(app)
+      }
+    })
 
-  const baseApps = apps.find(({ type }) => type === 'base')
-  const others = apps.filter((category) => category !== baseApps)
+    return result
+  }, [apps])
 
-  const { appsButton, contentWrapper, category, appList, link, footer } = appLauncher({ enableNew })
+  const styles = useMemo(() => {
+    const { appsButton, contentWrapper, category, appList, link, footer } = appLauncher({
+      enableNew,
+    })
+
+    return {
+      appsButton: appsButton(),
+      contentWrapper: contentWrapper(),
+      category: category(),
+      appList: appList(),
+      link: link(),
+      footer: footer(),
+    }
+  }, [enableNew])
 
   return (
     <Dropdown {...props}>
-      <DropdownTrigger>
-        <Button
-          prefix={enableNew ?? <FaToolboxIcon />}
-          suffix={enableNew ? <FaCaretDownIcon /> : undefined}
-          className={appsButton()}
-        >
-          {triggerLabel}
-        </Button>
-      </DropdownTrigger>
+      <MemoizedDropdownTrigger
+        enableNew={enableNew}
+        decorators={decorators}
+        className={styles.appsButton}
+      />
       <DropdownContent controllable>
         {/* eslint-disable-next-line smarthr/a11y-heading-in-sectioning-content */}
-        <Stack as="nav" gap={1.5} className={contentWrapper()}>
+        <Stack as="nav" gap={1.5} className={styles.contentWrapper}>
           <Stack gap={1.5}>
-            {baseApps && (
-              <Section>
-                <Stack gap={0.5} className={category()}>
-                  <Heading type="subSubBlockTitle">{baseApps.heading}</Heading>
-                  {/* eslint-disable-next-line smarthr/best-practice-for-layouts */}
-                  <Cluster as="ul" gap={1} className={appList()}>
-                    {appItems(baseApps.items, link())}
-                  </Cluster>
-                </Stack>
-              </Section>
+            {calculatedApps.base && (
+              <Stack gap={0.5} className={styles.category} as="section">
```

Comment:
`Section > Stack` 構造だったため、`Stack[as="section"]` に修正しています

Reply:
SectioningContentを使うようにするなら
```suggestion
              <Stack gap={0.5} className={styles.category} as={Section}>
```
でもいいかもと思いました (特に強い思いがあるわけではないです)

Reply:
Sectionで渡さなくても内部的にSectionに切り替えられるロジックになっているしこのままで行きます。